### PR TITLE
add aep info

### PR DESCRIPTION
**Pull request by leartulaj**

### CODE REVIEW

**Review by symbolpunk — APPROVED** [review]
LGTM

**Review by amarrazza — CHANGES_REQUESTED** [review]
we need to add addl context about L3s on existing arbitrum chains

File: arbitrum-docs/launch-orbit-chain/orbit-gentle-introduction.md [hunk 1]
```diff
@@ -138,19 +138,9 @@ Our small-but-mighty team is hard at work developing tools and patterns that mak
...
-2.  **Propose to the Arbitrum DAO**: With the launch of the Arbitrum DAO, the DAO was also given co-licensor rights to approve additional L2s on Ethereum that do not settle to a DAO-governed chain. To utilize this mechanism, you’ll need to [submit a proposal to the Arbitrum DAO](https://docs.arbitrum.foundation/how-tos/create-submit-dao-proposal), and the DAO will democratically decide whether or not to grant the license for your proposed L2 chain.
-```
+L3 Orbit chains that settle to DAO-governed chains like Arbitrum One and Nova can be permissionlessly deployed without restrictions. For Orbit chains that are L2s or L3s settling to Ethereum, the [Arbitrum Orbit Expansion program](https://forum.arbitrum.foundation/t/the-arbitrum-expansion-program-and-developer-guild/20722) creates a self-service path to launching. Via the permissionless profit-sharing model of the program, Orbit chains are granted the rights to make customized L2s or L3s that settle to Ethereum directly or other Ethereum L2s such as optimistic rollups, zk-rollups, optimiums, and validiums.
```
Comment (amarrazza) [inline arbitrum-docs/launch-orbit-chain/orbit-gentle-introduction.md:143]:
```suggestion
L3 Orbit chains that settle to DAO-governed chains like Arbitrum One and Nova can be permissionlessly deployed without restrictions. For Orbit chains that are settling to any other Ethereum chains, the [Arbitrum Orbit Expansion program](https://forum.arbitrum.foundation/t/the-arbitrum-expansion-program-and-developer-guild/20722) creates a self-service path to launching. Via the permissionless revenue-sharing model of the program, Orbit chains are granted the rights to build customized L2s or L3s (or L4s and beyond) that settle to Ethereum directly or other Ethereum chains, such as optimistic rollups, zk-rollups, optimiums, and validiums.
```

File: arbitrum-docs/launch-orbit-chain/orbit-gentle-introduction.md [hunk 2]
```diff
@@ -138,19 +138,9 @@ Our small-but-mighty team is hard at work developing tools and patterns that mak
...
-2.  **Propose to the Arbitrum DAO**: With the launch of the Arbitrum DAO, the DAO was also given co-licensor rights to approve additional L2s on Ethereum that do not settle to a DAO-governed chain. To utilize this mechanism, you’ll need to [submit a proposal to the Arbitrum DAO](https://docs.arbitrum.foundation/how-tos/create-submit-dao-proposal), and the DAO will democratically decide whether or not to grant the license for your proposed L2 chain.
-```
+L3 Orbit chains that settle to DAO-governed chains like Arbitrum One and Nova can be permissionlessly deployed without restrictions. For Orbit chains that are settling to any other Ethereum chains, the [Arbitrum Orbit Expansion program](https://forum.arbitrum.foundation/t/the-arbitrum-expansion-program-and-developer-guild/20722) creates a self-service path to launching. Via the permissionless revenue-sharing model of the program, Orbit chains are granted the rights to build customized L2s or L3s (or L4s and beyond) that settle to Ethereum directly or other Ethereum chains, such as optimistic rollups, zk-rollups, optimiums, and validiums.
```
Comment (amarrazza) [inline arbitrum-docs/launch-orbit-chain/orbit-gentle-introduction.md:143]:
```suggestion
L3 Orbit chains that settle to DAO-governed chains like Arbitrum One and Nova can be permissionlessly deployed without restrictions. For Orbit chains that are settling to any other Ethereum chain, the [Arbitrum Orbit Expansion program](https://forum.arbitrum.foundation/t/the-arbitrum-expansion-program-and-developer-guild/20722) creates a self-service path to launching. Via the permissionless revenue-sharing model of the program, Orbit chains are granted the rights to build customized L2s or L3s (or L4s and beyond) that settle to Ethereum directly or other Ethereum chains, such as optimistic rollups, zk-rollups, optimiums, and validiums.
```

**Review by amarrazza — APPROVED** [review]
lgtm

**Pull request merged**